docs/admin/kubelet.md

```diff
@@ -88,6 +88,7 @@ kubelet
       --event-burst=10: Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. Only used if --event-qps > 0
       --event-qps=5: If > 0, limit event creations per second to this value. If 0, unlimited.
       --eviction-hard="": A set of eviction thresholds (e.g. memory.available<1Gi) that if met would trigger a pod eviction.
+      --eviction-max-pod-grace-period=0: Maximum allowed grace period (in seconds) to use when terminating pods in response to a soft eviction threshold being met.
```

Comment:
You asked for 0 as the default?  In that manner a soft eviction threshold being met for a specified duration had the same eviction behavior as a hard eviction threshold.  Maybe we need a -1 to denote just use what the pod evicted specified.